source/options.html

```diff
@@ -4,99 +4,84 @@
 			<span id="validation"></span>
-		</label>
-	</p>
-	<ul>
-		<li data-validation data-scope="valid_token">The token enables <a href="https://github.com/refined-github/refined-github/search?q=github-helpers+api">some features</a> to <strong>read</strong> data from public repositories
-		<li data-validation data-scope="public_repo">The <code>public_repo</code> scope lets them <strong>edit</strong> your public repositories
-		<li data-validation data-scope="repo">The <code>repo</code> scope lets them <strong>edit private</strong> repositories as well
-		<li data-validation data-scope="delete_repo">The <code>delete_repo</code> scope is only used by the <code>quick-repo-deletion</code> feature
-	</ul>
-
-	<hr>
-
-	<p>
-		<label>
-			<strong>🔗 Refined GitHub link</strong><br>
-			This page will open when clicking the toolbar icon
-			<input type="url" name="actionUrl" placeholder="https://github.com" spellcheck="false" autocomplete="off" autocapitalize="off">
-		</label>
-	</p>
+		</p>
+		<ul>
+			<li data-validation data-scope="valid_token">The token enables <a href="https://github.com/refined-github/refined-github/search?q=github-helpers+api">some features</a> to <strong>read</strong> data from public repositories
+			<li data-validation data-scope="public_repo">The <code>public_repo</code> scope lets them <strong>edit</strong> your public repositories
+			<li data-validation data-scope="repo">The <code>repo</code> scope lets them <strong>edit private</strong> repositories as well
+			<li data-validation data-scope="delete_repo">The <code>delete_repo</code> scope is only used by the <code>quick-repo-deletion</code> feature
+		</ul>
+	</details>
 
-	<hr>
+	<details id="features">
+		<summary><strong class="features-header">🔋 Features</strong></summary>
+		<p>
+				<input id="filter-features" type="text" placeholder="Find features" spellcheck="false" autocomplete="off" autocapitalize="off">
+		</p>
+		<div class="js-features"></div>
+	</details>
 
-	<p>
-		<label>
-			<strong>💅 Custom CSS</strong> (useful to undo unwanted styles)
-			<textarea name="customCSS" rows="2" spellcheck="false"></textarea>
-		</label>
-	</p>
+	<details id="css">
+		<summary><strong>💅 Custom CSS</strong></summary>
+		<p>Like a userstyle, useful to undo unwanted style changes</p>
+		<p><textarea name="customCSS" rows="2" spellcheck="false"></textarea></p>
+	</details>
 
-	<hr>
+	<details id="action">
+		<summary><strong>🔗 Refined GitHub link</strong></summary>
+		<p>This page will open when clicking the toolbar icon</p>
+		<p><input type="url" name="actionUrl" placeholder="https://github.com" spellcheck="false" autocomplete="off" autocapitalize="off"></p>
+	</details>
 
-	<div>
-		<p id="show-debugging">
-			<button>Show debugging tools</button>
+	<details id="bisect">
+		<summary><strong>🔎 Identify feature</strong></summary>
```

Comment:
Extracted to its own section

<img width="550" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/221105419-b8af7bc3-68e0-4043-b042-c777a0c0b21a.png">